tests/baselines/reference/derivedClassSuperStatementPosition.js

```diff
@@ -73,6 +106,49 @@ var DerivedBasic = /** @class */ (function (_super) {
     }
     return DerivedBasic;
 }(Object));
+var DerivedAfterParameterDefault = /** @class */ (function (_super) {
+    __extends(DerivedAfterParameterDefault, _super);
+    function DerivedAfterParameterDefault(x) {
+        if (x === void 0) { x = false; }
+        var _this = this;
+        _this.x1 = x;
+        _this = _super.call(this, x) || this;
+        _this.x2 = x;
+        return _this;
+    }
+    return DerivedAfterParameterDefault;
+}(Object));
+var DerivedAfterRestParameter = /** @class */ (function (_super) {
+    __extends(DerivedAfterRestParameter, _super);
+    function DerivedAfterRestParameter() {
+        var x = [];
+        for (var _i = 0; _i < arguments.length; _i++) {
+            x[_i] = arguments[_i];
+        }
+        var _this = this;
+        _this.x1 = x;
+        _this = _super.call(this, x) || this;
+        _this.x2 = x;
+        return _this;
+    }
+    return DerivedAfterRestParameter;
+}(Object));
+var DerivedComments = /** @class */ (function (_super) {
+    __extends(DerivedComments, _super);
+    function DerivedComments() {
+        var _this = this;
+        // c1
+        console.log();
+        _this = 
+        // c2
+        _super.call(this) || this;

```

Comment:
This is still another odd comment placement. I think you need to set the comment range for the `_this = super.call(this) || this` expression to the original `super()` expression.

Reply:
The RWC tests are showing this comment behavior in a number of places as well.